### PR TITLE
Created parser for iosxe show snmp user

### LINKED ISSUE #85: 36. show snmp user
```
gw1# show snmp user
Load for five secs: 2%/0%; one minute: 1%; five minutes: 1%
Time source is NTP, 17:00:21.178 GMT Tue Sep 8 2020


``` 
 ```
gw2# show snmp user
User name: SNMPv3-alfa
Engine ID: 800000090300002790FBCF00
storage-type: nonvolatile	 active
Authentication Protocol: SHA
Privacy Protocol: AES128
Group-name: ALFA

``` 
 ```
gw3# show snmp user
User name: SNMPv3-alfa
Engine ID: 80000009030084BA8DA14100
storage-type: nonvolatile	 active
Authentication Protocol: SHA
Privacy Protocol: AES128
Group-name: ALFA

``` 
 ```
gw4# show snmp user
Load for five secs: 4%/0%; one minute: 4%; five minutes: 4%
Time source is NTP, 16:00:41.528 PDT Tue Sep 8 2020


User name: SNMPv3-alfa
Engine ID: 80000009030084BA8DA12F80
storage-type: nonvolatile	 active
Authentication Protocol: SHA
Privacy Protocol: AES128
Group-name: ALFA

```

### PR DESCRIPTION
**Pull request by qduk**

### CODE REVIEW

**Comment by qduk** [comment]
Did a PR against branch whitelist per recommendation of @reustaquio22.

**Comment by reustaquio22** [comment]
I just realized those changes were merged. https://github.com/CiscoTestAutomation/genieparser/pull/227
Once networktocode gets the new updates from cisco's repo, you should be able to do a PR against master

File: src/genie/libs/parser/iosxe/show_snmp.py [hunk 1]
```diff
@@ -79,3 +79,139 @@ def cli(self, output=None):
...
+        else: 
+            output = output
+
+        # User name: SNMPv3-alfa
+        # Engine ID: 800000090300002790FBCF00
+        # storage-type: nonvolatile	 active
+        # Authentication Protocol: SHA
+        # Privacy Protocol: AES128
+        # Group-name: ALFA
+        #
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/show_snmp.py]: You don't need to copy the whole thing, I would just do one with all of the options. In this case, that is with and without whitelist.

Comment (itdependsnetworks) [reply src/genie/libs/parser/iosxe/show_snmp.py]: Once you do this, please put in a PR to Cisco's branch. After that, make sure to link the Cisco PR in the comments of this PR.

**Comment by qduk** [comment]
[PR](https://github.com/CiscoTestAutomation/genieparser/pull/236) put into Cisco's branch

**Pull request merged**